VnE/Source/Extensions/ChemPack/voronoi.py

```
@@ -0,0 +1,132 @@
+#  ******************************************************************************************
+
+
+def execute():
+    from . import MAIN_WIDGET, MOLECULE_SYSTEMS, TREE_MODEL
+    from ... import point_class
+    from PySide6.QtOpenGLWidgets import QOpenGLWidget
+    from ..ChemPack.ui.select_mol_dialog import SelectMolDialog
+    import numpy as np
+    import cpplib
+
+    def fracToDec(a, b, c, al, be, ga, coords):
```

Ruff (0.14.14)

[warning] Missing return type annotation for private function `fracToDec`

(ANN202)